test/DatabaseIntegrationTest/Databases/Users/AdminUser.sql

```diff
@@ -0,0 +1,6 @@
+BEGIN TRANSACTION;
+
+INSERT INTO User (id, name, surname, login, password, role, creationTime, updateTime)
+VALUES ("73A800FC-621C-4D94-A1A4-0B39075458D4", "John", "Smith", "jsmith",	"P@assw0rd", 1, "20170212T093045", "20190115T152318");
```

Comment:
Shouldn't we be using encrypted-like values for password?

Reply:
We should, but at this point we aren't doing it yet. I would suggest opening an issue for that because this is something that already happens in master and it does not belong to DB tests.